_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,329 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are instruments to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML element][] that has a [visible][] [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed:** The `innerText` property of the [element][html element] changes multiple times within a 10 minute time span where there is no [user interaction][]; and
+- **no child changed:** the [element][html element] does not have [children][child] in the [flat tree][] whose `innerText` property also changes; and
+- **not alone:** the [element][html element] has an [ancestor][] element in the [flat tree][] with a non-empty `innerText` property whose value is different from the `innerText` of the test target.
+
+**Note:** The test targets for this rule are elements that have their text content change multiple times, on a page with further content, if that change happens without user intervention.
+
+**Note:** The 10 minute time span is an arbitrary limit which is not included in WCAG. Content that updates less frequently, or that doesn't update on a regular interval, will be inapplicable for this rule but may nonetheless fail [Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]. It is however a much lesser accessibility issue, and without any time limit this rule would be virtually untestable.
+
+## Expectation 1
+
+For each test target there exists a set of [instruments][instrument] to achieve one of the following objectives:
+
+- pause and resume the change of the [visible text content][]; or
+- stop the change of the [visible text content][]; or
+- hide the content that changes; or
+- alter the frequency of the changes of the [visible text content][].
+
+**Note:** If there is more than one test target, the same [instrument][] may be used to pause, stop, hide or alter the frequency of several or even all test targets.
+
+## Expectation 2
+
+Each [instrument][] in the set of [instruments][instrument] from Expectation 1 is in one of the following:
+
+- the same [web page](#web-page-html) of the test target; or
+- the same [web page](#web-page-html) of the test target as a result of an action from the user; or
```

Comment:
I think this is simultaneously too restrictive and to open-ended. Too restrictive because it can only ever be a single "action" away. And too open-ended because "what is an action"? How do you test this consistently?